<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionOptionsWidgetClass</class>
 <widget class="QWidget" name="ConnectionOptionsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>89</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdtProxy" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Host:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="buddy">
        <cstring>lneHost</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lneHost"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="buddy">
        <cstring>spbPort</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>5222</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblSslProtocol">
       <property name="text">
        <string>SSL Protocol:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSslProtocol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCertCheckMode">
       <property name="text">
        <string>Certificate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbCertCheckMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chbUseLegacySSL">
     <property name="text">
      <string>Use legacy SSL connection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
